vega_sim/scenario/ideal_market_maker_v2/agents.py

```diff
@@ -300,7 +339,7 @@ def _place_or_amend_order(
                 side=side,
                 volume=volume,
                 order_type=vega_protos.Order.Type.TYPE_LIMIT,
-                wait=False,
+                wait=True,
```

Comment:
Why are we now waiting here? Does this slow things down?

Reply:
Just set 'True' to make sure the order submitted successfully, most of the time will use instead amend_order function, so no big impact on the running time.